setup.cfg

```diff
@@ -23,6 +23,8 @@ addopts =
 
 filterwarnings =
     ignore:the matrix subclass:PendingDeprecationWarning
+    # The following warning is raised when PyArrow is not installed on pandas 2.2
+    ignore:Pyarrow will become a required dependency of pandas:DeprecationWarning
```

Comment:
We merged something in skrub 2 days ago. If this is regex, you need to add `(?s).*` at the beginning because this is a multiline string and `.*` at the end.

Reply:
FYI: https://github.com/skrub-data/skrub/pull/880/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R48-R49

Reply:
Not 100% sure whether this is a regex or not. According to sure https://docs.python.org/3/library/warnings.html#the-warnings-filter it depends:

> message is a string containing a regular expression that the start of the warning message must match, case-insensitively. In [-W](https://docs.python.org/3/using/cmdline.html#cmdoption-W) and [PYTHONWARNINGS](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONWARNINGS), message is a literal string that the start of the warning message must contain (case-insensitively), ignoring any whitespace at the start or end of message.

We might be in the -W or PYTHONWARNINGS case depending on how pytest filterwarnings gets passed down by pytest? needs to be checked in `test_script.sh` this looks more like `-W`, in `setup.cfg` not sure.